COMPILING.md

```diff
@@ -21,11 +21,11 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
    The GNU Make needs to be version 3.81 or higher.
    On Debian-like distributions, do
    ```
-   apt-get install g++ gcc flex bison make git libwww-perl patch
+   apt-get install g++ gcc flex bison make git libwww-perl patch openjdk-7-jdk
```

Comment:
(Just a note for me to revert these changes as part of https://github.com/diffblue/cbmc/issues/1869).